### PR TITLE
feat: update to spago@next

### PR DESCRIPTION
**Pull request by srghma**
**Description of the change**

update to spago@next

locally tests dont work, but on ci - work

---

**Checklist:**

- [ ] Added the change to the changelog's "Unreleased" section with a link to this PR and your username
- [ ] Linked any existing issues or proposals that this pull request should close
- [ ] Updated or added relevant documentation in the README and/or documentation directory
- [ ] Added a test for the contribution (if applicable)

### CODE REVIEW

File: packages.dhall_
Comment (thomashoneyman) [inline packages.dhall_:1]: What’s the reason to rename these dhall files? If we’re keeping them around for spago legacy users then we should keep the old names. If we don’t need them anymore we should delete them. cc @colinwahl

Comment (srghma) [reply packages.dhall_]: removed, tnx

**Comment by natefaubion** [comment]
I believe latest spago has landed on camelCase fields instead snake_case, FWIW.

**Comment by srghma** [comment]
> I believe latest spago has landed on camelCase fields instead snake_case, FWIW.

again update everything

```
sd -s "packageSet" "package_set" $(fd --type file -e yaml -e yml -e purs)
sd -s "extraPackages" "extra_packages" $(fd --type file -e yaml -e yml -e purs)
# or 
spago build --migrate ?
```

NB: not yet published to npm

**Comment by thomashoneyman** [comment]
Looks like the spago in CI is old? And — we don’t need psa anymore if we use the new spago, it’s bundled in.

File: .github/workflows/ci.yml
Comment (thomashoneyman) [inline .github/workflows/ci.yml]: Honestly we could just ditch setup-purescript here and install purs and spago via npm so the libraries aren’t relying on each other.

Comment (srghma) [reply .github/workflows/ci.yml]: ok, fixed

This workflow is awaiting approval from a maintainer in https://github.com/purescript-contrib/purescript-github-actions-toolkit/pull/20

**Review by thomashoneyman — APPROVED** [review]

**Pull request merged**
